Cargo.toml

```diff
@@ -9,32 +9,43 @@ repository = "https://github.com/QuantumFusion-network"
 resolver = "2"
 
 members = [
-    "runtime",
-    "node",
-    "pallets/qf-polkavm-dev",
-    "qf-test-runner"
+    "runtimes/solochain/runtime",
+    "runtimes/solochain/node",
```

Comment:
Node crates are not a part of the runtime, let's keep node directory in the repository root. Please check template or other substrate-based projects for the directory layout.